### PR TITLE
cdc: Add storage sink pipeline

### PR DESCRIPTION
**Pull request by CharlesCheung96**

### CODE REVIEW

File: pipelines/pingcap/tiflow/latest/pull_cdc_integration_storage_test.groovy [hunk 1]
```diff
@@ -0,0 +1,186 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+// should triggerd for master branches
+@Library('tipipeline') _
+
+final K8S_NAMESPACE = "jenkins-tiflow"
+final GIT_FULL_REPO_NAME = 'pingcap/tiflow'
+final GIT_CREDENTIALS_ID = 'github-sre-bot-ssh'
+final GIT_CREDENTIALS_ID2 = 'github-pr-diff-token'
+final POD_TEMPLATE_FILE = 'pipelines/pingcap/tiflow/latest/pod-pull_cdc_integration_test.yaml'
```
Comment (purelind) [inline pipelines/pingcap/tiflow/latest/pull_cdc_integration_storage_test.groovy:10]: It seem pod template name is not correct

Comment (CharlesCheung96) [reply pipelines/pingcap/tiflow/latest/pull_cdc_integration_storage_test.groovy]: PTAL

**Review by purelind — APPROVED** [review]
/lgtm

**Comment by CharlesCheung96** [comment]
ref https://github.com/pingcap/tiflow/issues/8826

**Pull request merged**